ci/scripts/website_build.sh

```diff
@@ -85,18 +84,29 @@ main() {
     fi
     echo "Latest version: ${latest_version} in directory ${latest_dir}"
 
+    # Make a copy of the latest release under a stable URL
+    rm -rf "${site}/current/"
+    cp -r "${site}/${latest_dir}" "${site}/current/"
+    git -C "${site}" add --force "current"
```

Comment:
```suggestion
    git -C "${site}" add -f "current"
```

Or we can use `--force` for other `git -C "#{site}" add ...`s.